src/ape/api/projects.py

```diff
@@ -240,6 +240,11 @@ class DependencyAPI(BaseInterfaceModel):
     A list of glob-patterns for excluding files in dependency projects.
     """
 
+    compiler_settings: Dict = {}
```

Comment:
Was thinking that this could be `config` or `config_override` to show that it could allow config for other plugins, not necessarily just compilers (although they are the most relevant plugins to configure ofc)

Reply:
ya, it would work with any plugin so probably should use a better name